frontend/src/stores/redux/loggedUser/loggedUserSlice.ts

```diff
@@ -15,11 +15,14 @@ export const loggedUserSlice = createSlice({
     initialState,
     reducers: {
         signIn: (state, action: PayloadAction<ILoggedUser>) => ({...action.payload}),
+        addToken: (state, action: PayloadAction<string>) => {
+            state.token = action.payload;
```

Comment:
where do you use it?

Reply:
if i want to create `loggedUser `i need to get `user`, but if i want to get user i need to provide token, so i add this reducer for this

Reply:
not clear what you said

Reply:
forgot to add to call this reducer in code (idk why i deleted it), so i will explain more detailed: 
1) when i get token i need to save it in store to get it in httpClient
2) when i call getUserService i used interceptor that add token in header from store
3) when i get user from server, i create loggedUser which will be contained in store

Reply:
correct version of fucntion `const handleSignIn = async () => {
        if (!isValid) return;
        const token = await logInService(authCredentials);
        if (token === "") return;
        dispatch(addToken(token));
        const decodedToken = decodeToken(token);
        const user = await getUserService(decodedToken.userId, decodedToken.userType);
        const loggedUser = createLoggedUser(token, user);
        if (loggedUser) {
            dispatch(signIn(loggedUser));
        }

    }`